src/data/real/ereal.lean

```diff
@@ -165,6 +165,39 @@ begin
   { simpa using hx },
 end
 
+lemma ereal.to_real' {x : ereal} (h : x ≠ ⊥) (h' : x ≠ ⊤) :
+  ∃ (c : ℝ), (c : ereal) = x := by {rcases x with rfl|rfl|x;tauto}
+
+lemma ereal.eq_top_iff (x : ereal) : x = ⊤ ↔ ∀ (y : ℝ), (y : ereal) < x :=
```

Comment:
```suggestion
lemma ereal.eq_top_iff_forall_lt (x : ereal) : x = ⊤ ↔ ∀ (y : ℝ), (y : ereal) < x :=
```
to match `enat.eq_top_iff_forall_lt`. I think you might be able to copy the approach of the proof there to make this one shorter too.